bot/src/main/resources/discord/appDirectory/description_fr.md

```diff
@@ -0,0 +1,21 @@
+Ce bot sert à lancer des dés dans discord.
+Le bot doit être configuré avec une commande slash dans un canal et fournit ensuite un message avec des boutons.
+En cliquant sur un bouton, le bot lancera l'expression de dé stockée pour le bouton, affichera le résultat et déplacera le message avec les boutons vers le bas du canal.
+Si le message est en attente, il sera seulement copié et non déplacé.
+Cela permet à l'utilisateur de lancer les dés sans avoir à taper de commandes et améliore ainsi la convivialité, en particulier pour les utilisateurs d'écrans tactiles.
+Le bot prend en charge les fils de discussion Discord (le message du bouton doit être ajouté après la création du fil de discussion), les forums, le texte en voix, et il est possible d'envoyer la réponse dans un canal différent.
```

Comment:
le texte en voix -> Text in Voice (keep english here, we're used to and translating it feels wrong)